src/app/components/task-button/task-button.component.ts

```diff
@@ -59,4 +62,13 @@ export class TaskButtonComponent implements OnInit {
       { display: 'inline-block' }
     );
   }
+  onTaskButtonClick({ dataJob, id }: TaskEventData): void {
+    switch (dataJob) {
+      case this.constantsService.DELETE_TODO:
+        this.taskButtonClick.emit({ id, dataJob });
+        break;
+      default:
+        break;
```

Comment:
I would not keep the default if it is not doing anything.

Reply:
done!